sympy/utilities/runtests.py

```diff
@@ -671,6 +671,7 @@ def _doctest(*paths, **kwargs):
         "sympy/physics/unitsystems.py", # raises deprecation warning
         "sympy/parsing/latex/_antlr/latexlexer.py", # generated code
         "sympy/parsing/latex/_antlr/latexparser.py", # generated code
+        "sympy/integrals/rubi/rubi.py"
```

Comment:
are we blacklisting the doctests as well?

Reply:
only for `rubi.py`, not for utility_functions. 
Actually, the loading time is around 10 - 11 minutes. So, travis fails. If nothing happens in 10 minutes, travis fails. So I have blacklisted it.